ls1intum/Artemis
# `Development`: Update Apollon to v2.12.7

### Checklist
#### General


### Motivation and Context
Update version of Apollon from [v2.12.6](https://github.com/ls1intum/Apollon/releases/tag/v2.12.6) to [v2.12.7](https://github.com/ls1intum/Apollon/releases/tag/v2.12.7)

### Description
This bugfix release of Apollon fixes the alignment of elements to grid issue. 
As Apollon only supports scaling factors with numbers having one decimal place (0.7, 0.8, 1,...) the scaling factor is now changed from 0.75 to 0.8.

**Changes include:**
- Set the default scale factor of Apollon to 0.8 instead of 0.75
- Set the moving/resizing factor of elements to 10 (previous value: 20)
- Fix alignment issue with following elements:
1. Rectangle in syntax tree 
2. Fork nodes
3. Terminal, Process and Function Call in Flow chart


### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create a modeling exercise
2. Observe all the elements (especially the one mentioned above) are aligned perfectly with the grid



### Screenshots
#### Before
![204931063-27d85340-bc35-4898-b6ab-a7ea03638146](https://user-images.githubusercontent.com/14681902/207728658-6fc04428-cfdb-496c-b6a8-61d45b5f61b5.png)

#### After
![204930662-d9abd526-aa84-4e44-8ab9-700ccc9bec97](https://user-images.githubusercontent.com/14681902/207728692-f37dc67d-4b42-4123-8ffd-97904798baa7.png)
